packages/component-library/stories/Icons.ts

```diff
@@ -0,0 +1,379 @@
+import department from "@kaizen/component-library/icons/department.icon.svg"
+import informationReport from "@kaizen/component-library/icons/exclamation-octagon.icon.svg"
+import percentage from "@kaizen/component-library/icons/percentage.icon.svg"
+import branching from "@kaizen/component-library/icons/branching.icon.svg"
+import competencyLibrary from "@kaizen/component-library/icons/competency-library.icon.svg"
+
+// USERS
+import individual from "@kaizen/component-library/icons/person.icon.svg"
+import user from "@kaizen/component-library/icons/user.icon.svg"
+import users from "@kaizen/component-library/icons/users.icon.svg"
+import userAdd from "@kaizen/component-library/icons/user-add.icon.svg"
+import userSelect from "@kaizen/component-library/icons/user-select.icon.svg"
+import userDelete from "@kaizen/component-library/icons/user-delete.icon.svg"
+import userSettings from "@kaizen/component-library/icons/user-settings.icon.svg"
+import userExit from "@kaizen/component-library/icons/user-exit.icon.svg"
+import userUpdate from "@kaizen/component-library/icons/user-update.icon.svg"
+
+// SURVEYS
+import surveys from "@kaizen/component-library/icons/surveys.icon.svg"
+import surveyDraft from "@kaizen/component-library/icons/draft.icon.svg"
+import surveyOpen from "@kaizen/component-library/icons/open.icon.svg"
+import surveyClosed from "@kaizen/component-library/icons/closed.icon.svg"
+import surveyArchived from "@kaizen/component-library/icons/archived.icon.svg"
+import surveyBlank from "@kaizen/component-library/icons/blank.icon.svg"
+import surveyTemplate from "@kaizen/component-library/icons/template.icon.svg"
+import surveyUnattributed from "@kaizen/component-library/icons/unattributed.icon.svg"
+
+// VIEWS
+import gridView from "@kaizen/component-library/icons/grid-view.icon.svg"
+import listView from "@kaizen/component-library/icons/list-view.icon.svg"
+import visible from "@kaizen/component-library/icons/visible.icon.svg"
+import invisible from "@kaizen/component-library/icons/invisible.icon.svg"
+import hierarchy from "@kaizen/component-library/icons/hierarchy.icon.svg"
+import navigator from "@kaizen/component-library/icons/navigator.icon.svg"
+
+// DIRECTIONAL
+import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
+import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import arrowDown from "@kaizen/component-library/icons/arrow-down.icon.svg"
+import arrowUp from "@kaizen/component-library/icons/arrow-up.icon.svg"
+import chevronDown from "@kaizen/component-library/icons/chevron-down.icon.svg"
+import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
+import sortDescending from "@kaizen/component-library/icons/sort-descending.icon.svg"
+import sortAscending from "@kaizen/component-library/icons/sort-ascending.icon.svg"
+// import reverse from "@kaizen/component-library/icons/reverse.icon.svg"
```

Comment:
Why is this one (and some other ones) commented out?

Reply:
They were ones that were missing from the component lib but are in figma - i have removed and made note.

Reply:
Great, is that "note" as in "card in Jira"? ;) If not, could you please put one up for those, thanks!